openslides/assignments/views.py

```diff
@@ -497,32 +497,31 @@ def append_to_pdf(self, story):
                     cell.append(Paragraph(
                         "(%s)" % candidate.structure_level,
                         stylesheet['Ballot_option_suffix_YNA']))
+                if self.poll.yesnoabstain:
```

Comment:
the change of indentation on the following lines fixes an issue that haven't been seen before: if a candidate had a structure_level, they were printed without ballot options on the paper.